src/Core/Domain/OrderReturnState/OrderReturnStateSettings.php

```diff
@@ -24,12 +24,14 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-header('Expires: Mon, 26 Jul 1997 05:00:00 GMT');
```

Comment:
Careful the `index.php` should also be reverted and kept along with the removed templates